.gitmodules

```diff
@@ -19,3 +19,6 @@
 [submodule "third-party/gflags"]
 	path = third-party/gflags
 	url = https://github.com/gflags/gflags.git
+[submodule "third-party/zenfs"]
```

Comment:
This is not needed anymore?

By the way, I would recommend putting the [license](https://github.com/westerndigitalcorporation/zenfs/blob/master/LICENSE.Apache) of `zenfs` at the root of `env`.

Reply:
@skyzh thanks, I've updated the patch. Since we didn't follow the Github workflow seriously before, we have to rebase all history commits, this is not the last patch yet.